ee/vellum_ee/workflows/display/utils/expressions.py

```diff
@@ -366,6 +366,23 @@ def serialize_value(display_context: "WorkflowDisplayContext", value: Any) -> Js
             },
         }
 
+    if isinstance(value, MCPServer):
```

Comment:
[q] Does this not happen by default as part of `if isinstance(value, BaseModel)` below?